lib/lightspeed/request.rb

```diff
@@ -13,33 +15,32 @@ def self.verbose?
       !! @verbose
     end
 
-    def self.base_url
-      "https://api.merchantos.com/API"
+    def self.base_host
+      "api.merchantos.com"
+    end
+
+    def self.base_path
+      "/API"
     end
 
     def initialize(client, method:, path:, params: nil, body: nil)
+      @method = method
```

Comment:
I guess this is a sign I shouldn't comment on PRs when tired... looking again today I think it's fine. Sorry for the confusion.